### DISCUSSION TITLE
`master` branch has changed

### DISCUSSION THREAD

**Discussion by dgdavid** [agama-project/agama#1186] [Announcements]
Back in [January 24th](https://github.com/openSUSE/agama/discussions/1000#discussioncomment-8232103), it was decided to stop using Cockpit. As a result, the [`architectural_2024`](https://github.com/openSUSE/agama/tree/architecture_2024) branch was created to start the migration in parallel to development of `master`.

At the beginning of this week, such a branch [was merged into `master`](https://github.com/openSUSE/agama/pull/1177). Because it is not fully ready, I created a [`master-on-cockpit`](https://github.com/openSUSE/agama/tree/master-on-cockpit) feature branch to enable working in the UI with the Cockpit infrastructure until `master` is fully functional again. That's when I realized

* There was a great (and now lost) opportunity for us to start using `main` instead of `master`.
* We did the merge without announcing it, which may cause confusion among users/developers outside the team. @balsa-asanovic comes to my mind.

While the README contains almost all the necessary information to make the project work again, it's still hard to guess for someone who just run a `git pull` and suddenly has the environment messed up. Moreover, the user/developer can be confused without knowing that the storage part is now WIP.

**Comment by balsa-asanovic** [comment]
Thanks for the mention, good to know.

> it's still hard to guess for someone who just run a git pull and suddenly has the environment messed up.

This could of easily happened 😄

**Reply by dgdavid** [reply]
Please, do not hesitate asking for help if you cannot run the project with the new architecture. Or even let us know if you spot something to improve in the documentation or whatever worth to mention for help developers to make it work easily.

Thanks in advance.

**Comment by ancorgs** [comment]
All documentation is now updated and everyone involved in the development is aware of the changes, let's close this.
